omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -25,7 +43,39 @@ else
     PREFIX="/usr"
 fi
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
+binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+
+hab_target_path="$PREFIX/bin/hab"
+if [ -L "$hab_target_path" ]; then
+  loc=$(readlink "$hab_target_path")
+  case $loc in
+    *chef-workstation*)
+      # We own this path, so we're not taking it over from another install.
+      ;;
+    *)
+      takeover="1"
+  esac
+elif [ -f "$hab_target_path" ]; then
+  takeover="1"
+  rename="1"
+fi
+
+if [ "$takeover" = "1" ]; then
+  rm -rf "$hab_target_path"
+  echo ""
+  echo "NOTE: Chef Habitat on this node is now managed by Chef Workstation."
+  echo "      Future Workstation updates will always include the latest stable"
+  echo "      Habitat release."
+  echo ""
+fi
+
+if [ "$rename" = "1" ]; then
+  mv "$hab_target_path" "$hab_target_path.orig"
+  echo "'$hab_target_path' has been renamed to '$hab_target_path.orig'"
+  echo ""
```

Comment:
```suggestion
  echo ""
  echo "NOTE: Chef Habitat on this node is now managed by Chef Workstation."
  echo "      Existing install $hab_target_path' has been renamed to"
  echo "      '$hab_target_path.orig'"
  echo ""
```

Reply:
For context, the 'now managed' message will be displayed already;  this is an additional info message that wil be shown after that. 

I've changed the wording of the rename message in the commit coming up though, `Original habitat binary has been moved to '$hab_target_path.orig'"